modules/virt-defining-a-watchdog.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog"]
+= Defining a watchdog device
+
+.Procedure
+
+. Choose the watchdog action. This defines how the watchdog proceeds when the operating system (OS) no longer responds.
+
++
+.Available actions
+[horizontal]
+`poweroff`:: The virtual machine (VM) powers down, and the guest OS cannot react.
+`reset`:: The VM reboots in place, and the guest OS cannot react. Because the length of time required for the guest OS to reboot can cause liveness probes to timeout, you should avoid their use.
+`shutdown`:: The VM powers off gracefully and becomes non-runnable.
```

Comment:
I find this statement confusing. To say a VM has become "non-runnable" implies that I cannot re-start it. Is that the case here?

Reply:
@stu-gott This language was pulled from another doc I believe re: the available actions. Do you have a suggestion for clarity?

Reply:
The path forward is not clear to me. Further re-reading of the 3 actions might imply with wording is entirely correct and intentional. Do you know who was involved in creating the document you referenced?

Reply:
@rmohr do you have insight into the particular wording for these three actions?

Reply:
@stu-gott @rmohr Although I know I pulled the "non-runnable" wording from somewhere else, I can't find where. Perhaps it was a fever dream. Happy to remove that and end this at powers off gracefully - just let me know.

Reply:
The thing that has me confused is that I can't understand a clear distinction between poweroff and shutdown.

Reply:
@stu-gott From my notes it appears the guest OS can react in shutdown, unlike poweroff. This is probably worth calling out for clarity. It also looks like it needs to be unlocked in shutdown but maybe not in poweroff? Rewrite suggestion: `The VM powers off gracefully if it is unlocked. The guest OS can react. `

Reply:
From what I observed during my testing,

a) poweroff instantly brings down the VM

b) Shutdown gracefully brings down the VM, by stopping all the services.

Though in both the cases a) and b) , the VM automatically got into running state when the watchdog process was killed.

This was when in the vm spec, it had spec.running = False

Reply:
ok. Thanks for clarifying @kbidarkar.

@sabrinajess I suggest rewording the line this way:

```
`shutdown`:: The VM powers off gracefully.
```